src/plugins/intel_cpu/tests/functional/single_layer_tests/convolution.cpp

```diff
@@ -220,7 +220,8 @@ TEST_P(ConvolutionLayerCPUTest, CompareWithRefs) {
 
     if (!priority.empty()) {
         // Skip tests for brgconv convolution where kernel size = 1x1
-        if (priority[0] == "brgconv_avx512" || priority[0] == "brgconv_avx512_amx") {
+        if (priority[0] == "brgconv_avx512" || priority[0] == "brgconv_avx512_amx" ||
+            priority[0] == "brgconv_avx2") {
```

Comment:
using `one_of(priority[0], "brgconv_avx512", "brgconv_avx512_amx", "brgconv_avx2")` may be concise.